cmd/lncli/commands.go

```diff
@@ -2311,7 +2311,22 @@ func updateChannelPolicy(ctx *cli.Context) error {
 		return err
 	}
 
-	printRespJSON(resp)
+	// Parse the response into the final json object that will be printed
+	// to stdout. At the moment, this filters out the raw txid bytes from
+	// each failed update's outpoint and only prints the txid string.
+	var listFailedUpdateResp = struct {
```

Comment:
Would be nice to have a `if len(resp.FailedUpdates)==0` print "all channels updated successfully", but not a blocking comment since you've done the hard yards on this PR!

Reply:
I'm not sure what you mean but I added a print command in rpcserver.go.

```
if len(failedUpdates) == 0 {
	rpcsLog.Info("All channels updated successfully")
}
```

Reply:
Sorry, was unclear here, I meant print it just on the CLI rather than printing an empty list. But it's just a little bit of UX-sugar so not vital!

Reply:
Alright, I'll leave it as is then. Thx for your review.